soc/arm/nxp_kinetis/k2x/Kconfig.soc

```diff
@@ -0,0 +1,54 @@
+# Kconfig - Kinetis K2X MCU line
+#
+# Copyright (c) 2018 Prevas A/S
+# Copyright (c) 2019 bytes at work AG
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+choice
+prompt "Kinetis K2x MCU Selection"
+depends on SOC_SERIES_KINETIS_K2X
+
+config SOC_MK22FA12
```

Comment:
Does this need SMC to get HSRUN mode? I ended up having to add HAS_MCUX_SMC here and in the hal for myself to get the full speed clock

Reply:
The SOC I use (fx512) does not support HSRUN, so I dropped the SMC support. But for other MK22 I think you need it.

Reply:
Good to know, I'll add that when I add MK22FN512